moveit_core/robot_model/src/joint_model_group.cpp

```diff
@@ -554,6 +561,14 @@ void JointModelGroup::setDefaultIKTimeout(double ik_timeout)
   for (KinematicsSolverMap::iterator it = group_kinematics_.second.begin(); it != group_kinematics_.second.end(); ++it)
     it->second.default_ik_timeout_ = ik_timeout;
 }
+// Re-added this as its needed to compile robot_state, this has been removed on
+// https://github.com/ros-planning/moveit/pull/1288/files
```

Comment:
This function is not needed as long as the latest changes from moveit1 are synced back in

Please remove, this is a regression of feature set

Reply:
After https://github.com/ros-planning/moveit2/pull/47 and https://github.com/ros-planning/moveit2/pull/53, this removal indeed made sense.

Requested change applied at 77d2d9a398738c364e61cfb4ef53275bca41126a